### DISCUSSION TITLE
Contribution Graph | No work shown

### DISCUSSION THREAD

**Discussion by unassigned-shogun** [community/community#44392] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone and please pardon my ignorance on the subject matter.

I know this has come up a lot, but after much research, I still cannot seem to come to a viable solution to what I know is a simple matter. I have been working on a few projects over the past 3 months, but noticed that, while my **git pushes** do properly update onto the repository itself, by **contribution graph** shows no such updates. It gives the impression that I haven't done anything. The only thing the graph shows is when I create a repository and nothing more. Otherwise, It is just blank.

I work mainly via a **Virtual Machine** installation of **Zubuntu Linux** and made sure that my `git config user.mail` was properly set up. `git push` certainly works, as the GitHub repo itself shows such changes. I have my email set to private, thus rendering my commit email to **email.noreply.git.com**. And when I type `git config user.mail` on the CLI, I get the noreply email GitHub provides. When working on my projects, I use `git clone` to make a duplicate on my local machine and then `git add`, `git commit -m`, and `git push` for every change. Once more, the main repo itself shows the change, but nothing is recorded in the contribution graph.

Obviously, I am missing or overlooking something essential, and I can't seem to wrap my head around it. May I ask that you please help enlighten me as to the proper method of correcting this detail?

Best regards, 

https://github.com/The-Owl-Shogun

**Comment by lewisthagichu** [comment]
Hello @The-Owl-Shogun! Did you ever get a solution to this problem? I am also experiencing the same issue at the moment. Regards.
